grpc/server/dronecore_server.cpp.in

```diff
@@ -49,7 +49,11 @@ void RunServer()
     map_type map;
     std::string plugin;
     std::fstream file;
-    file.open("../proto/plugins.conf");
+    file.open("plugins.conf");
```

Comment:
This depends on where you call `dronecore_server` from, right? Couldn't we do something more like, for instance, `file.open("${PROJECT_SOURCE_DIR}/grpc/proto/plugins.conf");`?

Reply:
Or what about opening `./plugins.conf` by default, but also having it as a parameter?

Reply:
Well, I'm actually realizing it's not part of this "fix" PR. :-)